modules/iam-identity/src/main/java/com/ibm/cloud/platform_services/iam_identity/v1/model/SetProfileIdentityOptions.java

```diff
@@ -0,0 +1,334 @@
+/*
+ * (C) Copyright IBM Corp. 2023.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.cloud.platform_services.iam_identity.v1.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.ibm.cloud.sdk.core.service.model.GenericModel;
+
+/**
+ * The setProfileIdentity options.
+ */
+public class SetProfileIdentityOptions extends GenericModel {
+
+  /**
+   * Type of the identity.
+   */
+  public interface Type {
+    /** user. */
+    String USER = "user";
+    /** serviceid. */
+    String SERVICEID = "serviceid";
+    /** crn. */
+    String CRN = "crn";
+  }
+
+  /**
+   * Type of the identity.
+   */
+  public interface NewType {
+    /** user. */
+    String USER = "user";
+    /** serviceid. */
+    String SERVICEID = "serviceid";
+    /** crn. */
+    String CRN = "crn";
+  }
+
+  protected String profileId;
+  protected String type;
+  protected String newIdentifier;
+  protected String newType;
```

Comment:
The presence of these properties with the "new" prefix indicates that there is a name collision between the "type" parameter, and the "type" property within the requestBody schema for the "set_profile_identity" operation.
You'll need to rename (within the API definition) either the "type" parameter or the "type" schema property to avoid the collision.